src/scipp/plotting/widgets.py

```diff
@@ -30,7 +30,7 @@ def __init__(self, *, dims, formatters, ndim, dim_label_map, masks, sizes):
         # all_masks_button: button to hide/show all masks in a single click
         self.all_masks_button = None
 
-        self._slider_dims = dims[:len(dims) - ndim]
+        self._slider_dims = list(dims)[:len(dims) - ndim]
```

Comment:
Any reason why you didn't do it like this?
```suggestion
        self._slider_dims = list(dims[:len(dims) - ndim])
```

Reply:
No reason. Should I change it?